metrics/cli/lib/gcloud/strings/gcloud_strings.dart

```diff
@@ -22,6 +22,13 @@ If you have already accepted the terms of the GCloud service, skip this step. Ot
 
 Once you are done, press the ENTER to continue:''';
 
+  static const String enterProjectName = '''
+Please, choose the GCloud project name that suits you best.
+Note: The GCloud project name can contain only letters, numbers, single quotes, hyphens, spaces, or exclamation points, and must be between 4 and 30 characters.
```

Comment:
Would it be great to add a link for more detailed information about project name constraints?